### PR TITLE
[FEATURE] DotNetCore 2 added as Target-Framework

### PR DESCRIPTION
**Pull request by ohunecker**
Added .NetCore 2 as Target-Framework and updated Newtonsoft.Json to lastest version

### CODE REVIEW

**Comment by abjerner** [comment]
@ohunecker thanks for creating this PR 👍 

As you haven't described much about why you made this change, could you please provide some more details about your reasons from doing so?

.NET Core builds on top of .NET Standard, so .NET Core should therefore already by indirectly supported. The package doesn't use features that only exist in .NET Core.

Skybrud.Essentials has a dependency for an older version of JSON.net - this is actually by design. There is a fair chance that JSON.net is already installed in whatever project Skybrud.Essentials, so the dependency therefore references an older version of JSON.net, but doesn't restrict you from using a newer version instead.

As an example, some of our work involves older systems, where other NuGet packages (that we're not in control of) have a version specific dependency for JSON.net preventing us from updating our own reference to JSON.net. 

So to summarize this, I don't see the reason to add .NET Core as a target framework, at it is indirectly supported as we target .NET Standard. But as haven't used the newer versions of .NET Core myself, I could be wrong - so I'd be willing to accept this change, if you can provide a good reason for doing so. I can however not approve the updated version requirement for JSON.net, as that may cause problems in our internal pipeline for older projects that we still need to support.

**Comment by ohunecker** [comment]
@abjerner sry for my missing describtion and tanks a lot for your work.
These changes remove the warnings, that the package was restored to using dotNet 4.6.1.

JSON.net was also updated with the changes to use dotNetCore, but is not necessary.

**Comment by abjerner** [comment]
@ohunecker I just had another look at this, and I'm not sure why you're seeing any warnings. Did your project perhaps start out as .NET 4.6.1? Would it be possible for you to share the warnings?

I tried creating two new console applications, one targeting .NET Core 2.0 and the other targeting .NET Core 2.2. For both I've tried installing Skybrud.Essentials, and the dependency file is updated with the following:

```
"Skybrud.Essentials/1.1.9": {
    "dependencies": {
        "Newtonsoft.Json": "9.0.1",
        "System.Xml.XPath.XDocument": "4.3.0"
    },
    "runtime": {
        "lib/netstandard1.3/Skybrud.Essentials.dll": {}
    }
}
```

It specifically mentions .NET Standard 1.3. I also don't get any warnings when building the two projects.

**Comment by abjerner** [comment]
I'm closing this PR as I can't reproduce any warnings, and because .NET Core should be indirectly supported as the package is already targeting .NET Standard.
